llvm/include/llvm/IR/VFABIDemangler.h

```diff
@@ -44,6 +44,7 @@ enum class VFParamKind {
 enum class VFISAKind {
   AdvancedSIMD, // AArch64 Advanced SIMD (NEON)
   SVE,          // AArch64 Scalable Vector Extension
+  RVV,          // RISC-V Scalable Vector Extension
```

Comment:
"Scalable Vector Extension" is the expansion of SVE. Should drop "Scalable" for RISC-V, it's not used the in spec.

Reply:
Removed "scalable"